pom.xml

```diff
@@ -120,19 +115,23 @@
         <maven-checkstyle-plugin.version>3.1.0</maven-checkstyle-plugin.version>
         <maven-jacoco-plugin.version>0.8.4</maven-jacoco-plugin.version>
         <failsafe.version>1.1.0</failsafe.version>
-        <spring-data-solr.version>4.0.9.RELEASE</spring-data-solr.version>
+        <spring-data-solr.version>4.1.0.RC2</spring-data-solr.version>
+        <solr.version>8.2.0</solr.version>
         <opengis.version>19.1</opengis.version>
         <javax.xml.soap.version>1.4.0</javax.xml.soap.version>
         <jackson.version>2.10.0.pr1</jackson.version>
         <spring.version>5.1.8.RELEASE</spring.version>
-        <spring-boot-starter.version>2.1.6.RELEASE</spring-boot-starter.version>
+        <spring-boot-starter.version>2.1.7.RELEASE</spring-boot-starter.version>
         <annotation.version>1.3.1</annotation.version>
         <reactive-streams.version>1.0.2</reactive-streams.version>
         <fabric8.docker.plugin.version>0.30.0</fabric8.docker.plugin.version>
         <project.report.output.directory>project-info</project.report.output.directory>
-        <owasp.version>5.2.0</owasp.version>
+        <owasp.version>5.2.1</owasp.version>
         <slf4j.version>1.7.26</slf4j.version>
         <gpg-plugin.version>1.6</gpg-plugin.version>
+        <org.eclipse-jetty.version>9.4.19.v20190610</org.eclipse-jetty.version>
+        <micrometer.version>1.1.5</micrometer.version>
+        <jackson-databind.version>2.9.9.3</jackson-databind.version>
```

Comment:
Why are we creating a new version for this instead of using jackson.version which is newer.

Reply:
Oops, didn't check for existing properties :D